tests/acceptance/helpers-test.js

```diff
@@ -1,76 +1,185 @@
 import { test } from 'qunit';
 import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
 import { visit, find } from 'ember-native-dom-helpers';
+import {
+  calendarSelect,
+  calendarCenter
+} from 'ember-power-calendar/test-support';
 
 moduleForAcceptance('Acceptance | helpers | calendarCenter');
 
-test('`calendarCenter` invokes the `onCenterChange` action of the target component inside the selector we pass it', async function(assert) {
+test('`calendarCenter` invokes the `onCenterChange` action of the target component inside the selector we pass it', async function(
+  assert
+) {
   await visit('/helpers-testing');
-  assert.equal(find('.calendar-center-1 .ember-power-calendar-nav-title').textContent.trim(), 'October 2013');
+  assert.equal(
+    find(
+      '.calendar-center-1 .ember-power-calendar-nav-title'
+    ).textContent.trim(),
+    'October 2013'
+  );
 
   await calendarCenter('.calendar-center-1', new Date(2013, 8, 3));
-  assert.equal(find('.calendar-center-1 .ember-power-calendar-nav-title').textContent.trim(), 'September 2013', 'The nav component has updated');
-  assert.ok(find('.calendar-center-1 [data-date="2013-09-01"]'), 'The days component has updated');
+  assert.equal(
+    find(
+      '.calendar-center-1 .ember-power-calendar-nav-title'
+    ).textContent.trim(),
+    'September 2013',
+    'The nav component has updated'
+  );
+  assert.ok(
+    find('.calendar-center-1 [data-date="2013-09-01"]'),
+    'The days component has updated'
+  );
 });
 
-test('`calendarCenter` invokes the `onCenterChange` action of the target component WITH the selector we pass it', async function(assert) {
+test('`calendarCenter` invokes the `onCenterChange` action of the target component WITH the selector we pass it', async function(
```

Comment:
I think prettier auto formatted these following eslint rules. If you want I'll try to revert it